R/getAbundanceSample.R

```diff
@@ -0,0 +1,76 @@
+#' Returns abundance values for all \dQuote{FeatureID} in a \dQuote{SampleID}
+#'
+#' Basic extractor function for extracting \dQuote{FeatureID} abundances
```

Comment:
These files should be merged into one file: `getAbundance`. Redundancies are minimised that way. Also make clear that these are just convenience wrappers around `assay(x,name)[row,col]`.